pkg/component/kubernetes/controllermanager/controllermanager.go

```diff
@@ -653,6 +653,8 @@ func (k *kubeControllerManager) computeCommand(port int32) []string {
 			"--authentication-kubeconfig=" + gardenerutils.PathGenericKubeconfig,
 			"--authorization-kubeconfig=" + gardenerutils.PathGenericKubeconfig,
 			"--kubeconfig=" + gardenerutils.PathGenericKubeconfig,
+			"--kube-api-qps=100",
```

Comment:
Should we rather disable client-side rate limiting at all in favor of priority and fairness as per sig-apimachinery recommendation? ref https://github.com/kubernetes-sigs/controller-runtime/pull/3119